docs/assets/js/search.js

```diff
@@ -23,24 +22,19 @@ function displaySearchResults( results, store ) {
 
       // Show some preview text under each search results item.
       let previewText = '';
-      let itemFieldValue;
-      let field;
+      const searchMatchWordFragment = Object.keys( results[i].matchData.metadata )[0];
+      const searchMatchFields = results[i].matchData.metadata[ searchMatchWordFragment ];
 
-      // Search for the first preview field that is populated.
-      const searchFields = store.fields;
-      for ( const fieldId in searchFields ) {
-        field = searchFields[fieldId];
-        // Exclude certain fields from being considered for the result preview.
-        if ( field === 'title' || field === 'url' ) {
-          continue;
-        }
-
-        itemFieldValue = item[field];
-        if ( typeof itemFieldValue !== 'undefined' && itemFieldValue !== '' ) {
-          previewText = itemFieldValue;
-          break;
-        }
-      }
+      // Remove fields that should never appear as the preview.
+      delete searchMatchFields.id;
+      delete searchMatchFields.title;
+
+      previewText = item[ Object.keys( searchMatchFields )[0] ];
+
+      const regex = new RegExp( results.searchTerm, 'gi');
+      previewText = previewText.replace( regex, function replace( match ) {
+        return '<mark>' + match + '</mark>';
```

Comment:
When you demo this at stand-up get the team's opinion on the default bright yellow color for `mark`. And 💯 for the semantic HTML.

Reply:
They saw. They said nothing.